src/deploy.cmd.js

```diff
@@ -271,7 +278,6 @@ class DeployCommand extends AbstractCommand {
 
     const followoptions = {
       method: 'POST',
-      json: true,
```

Comment:
I think the `json` option doesn't have a meaning in helix-fetch, right?

Reply:
It did not in 1.0

Reply:
It did. You could specify the JSON payload (plain JS object) of `POST`, `PUT`, `PATCH` requests via the `json` option. 

In v2 the `json` option is not supported anymore but you can do the same via the `body` option:
https://github.com/adobe/helix-fetch/blob/main/v2-UPGRADE-GUIDE.md#post-json

Primitive `body` values however (e.g. `true`, `42`) are not (yet) interpreted as JSON payload. I guess the `json: true` line here is a leftover from the time when we used the `request` module.